drake/systems/plants/collision/BulletHeightMapTerrain.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __BulletHeightMapTerrain_H__
+#define __BulletHeightMapTerrain_H__
```

Comment:
Illegal macro name. Should be something like `DRAKE_BULLET_HEIGHT_MAP_TERRAIN_H_`. (Can't use leading or double underscores; standard calls for CAPS.)